rust/docs/sync/UserManager.adoc

```diff
@@ -0,0 +1,232 @@
+[#_struct_UserManager]
+=== UserManager
+
+*Implements traits:*
+
+* `Clone`
+* `Debug`
+
+Provides access to all user management methods.
+
+[caption=""]
+.Fields
+// tag::properties[]
+[cols="~,~,~"]
+[options="header"]
+|===
+|Name |Type |Description
+a| `connection` a| `Connection` a| 
+|===
```

Comment:
This should be hidden! (technical debt)
I believe there is a rustdoc "ignore" directive or smth like that.

Reply:
Removed (it is `#[doc(hidden)]` directive)